DSpace/DSpace
# Fix for #9589 Incorrect Export of Search Results with Double Quotation Marks in DSpace 7.

## References
* Fixes #9589

## Description
When exporting search results in DSpace 7 that include double quotation marks in the search query, the export does not reflect the original search query with double quotation marks. Instead, it exports the results of the search query without the double quotation marks.

## Instructions for Reviewers
Export the search results after performing a search in DSpace using a query that includes double quotation marks, for example: "search term". The export of search result bringing extra results in it which are not there in the result of original double quoted search query results.

## List of changes in this PR:
* Added minor change inside the parse method under DSpaceRunnable class to set StripLeadingAndTrailingQuotes as false in default parser builder.

## Guidance for how to test or review this PR 
1. Login with any user
1. Perform a search in DSpace using a query that includes double quotation marks, for example: "dengue vaccination".
2. Export the search results by clicking on Export search results as CSV.
3. Observe the export does not reflect the original search query result and instead exports the results without the double quotation marks also.
